vignettes/toInteger.Rmd

```diff
@@ -21,15 +21,29 @@ This has a couple of implications on design characteristics:
 
 This document goes through examples to demonstrate the calculations.
 The new function as of July, 2023 is the `toInteger()` which operates on group sequential designs to convert to integer-based total sample size and event counts at analyses.
-We begin with an abbreviated example for a time-to-event endpoint design to demonstrate basic concepts.
+As of November, 2024 the rounding defaults are changing as documented below.
+We begin with a summary of the method.
+Then we provide an abbreviated example for a time-to-event endpoint design to demonstrate basic concepts.
 We follow with a more extended example for a binary endpoint to explain more details.
 
+## Summary of method
+
+- With defaults, `gsSurv() |> toInteger(ratio = 0, roundUpFinal = TRUE)` 
```

Comment:
Could you please confirm if the value of `ratio` should be `0`?

Reply:
Yes! This changes default to just rounding, independent of sample size ratio specified. This matches EAST convention.